libs/wingsdk/src/shared-aws/function.ts

```diff
@@ -7,6 +7,16 @@ import { INFLIGHT_SYMBOL } from "../core/types";
 import { IInflightHost, Node, Resource } from "../std";
 import * as ui from "../ui";
 
+export const externalLibraries = [
+  "@aws-sdk/client-sso",
+  "@aws-sdk/client-sso-oidc",
+  "@aws-sdk/credential-provider-ini",
+  "@aws-sdk/credential-provider-process",
+  "@aws-sdk/credential-provider-sso",
+  "@aws-sdk/credential-provider-web-identity",
+  "@aws-sdk/token-providers",
+];
```

Comment:
Does esbuild's `external` option work with `*`?

```suggestion
export const externalLibraries = ["@aws-sdk/*"];
```

Even if so, maybe there's value in being explicit 🤷

Reply:
Either is ok to me